packages/authorization-process/src/model/domain/errors.ts

```diff
@@ -79,3 +81,25 @@ export function purposeIdNotFound(
     title: "Purpose id not found",
   });
 }
+
+export function securityUserNotFound(
+  requesterUserId: UserId,
+  userId: UserId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Security user not found for consumer ${requesterUserId} and user ${userId}`,
+    code: "securityUserNotFound",
+    title: "security User not found",
+  });
+}
```

Comment:
following @taglioni-r's renaming request
```suggestion
export function userWithoutSecurityPrivileges(
  requesterUserId: UserId,
  userId: UserId
): ApiError<ErrorCodes> {
  return new ApiError({
    detail: `User ${userId} does not have security privileges for consumer ${requesterUserId}`,
    code: "userWithoutSecurityPrivileges",
    title: "user without security privileges",
  });
}
```